etc/httpd/global.conf

```diff
@@ -576,6 +576,7 @@ RewriteRule ^/(\w{2,3}(?:-\w{2}(?:-mac)?)?/)?thunderbird/([3-9]\d\.\d(?:a2|beta|
 RewriteRule ^/(\w{2,3}(?:-\w{2}(?:-mac)?)?/)?thunderbird/([3-9]\d\.\d(?:a2|beta|\.\d)?)/system-requirements(/)?$ /b/$1thunderbird/$2/system-requirements$3 [L,PT]
 RewriteRule ^/(\w{2,3}(?:-\w{2}(?:-mac)?)?/)?thunderbird/releases(/)?$ /b/$1thunderbird/releases$2 [L,PT]
 RewriteRule ^/(\w{2,3}(?:-\w{2}(?:-mac)?)?/)?thunderbird/releases/(.*)$ http://website-archive.mozilla.org/www.mozilla.org/thunderbird_releasenotes/en-US/thunderbird/releases/$2 [L,R=301]
+RewriteRule ^/(\w{2,3}(?:-\w{2}(?:-mac)?)?/)?thunderbird/latest/(.*)(/)?$ /b/$1thunderbird/latest/$2$3 [L,PT]
```

Comment:
There are 2 URLs incorrectly redirecting to archive, so `/latest/(.*)/?$` should work.